### ISSUE TITLE
Column header name not focusable with screen reader

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#10254]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=30831&p=156475#p156475)

"Hi after doing some accessibility testing on our implementation of the taskboard ive found an issue:

Cannot focus on the column header name so screen reader cant read out what column your focused on."
